proptest/src/test_runner/runner.rs

```diff
@@ -39,7 +39,9 @@ use crate::test_runner::rng::TestRng;
 const ENV_FORK_FILE: &'static str = "_PROPTEST_FORKFILE";
 
 const ALWAYS: u32 = 0;
-const SHOW_FALURES: u32 = 1;
+/// Verbose level 1 to show failures. In state machine tests this level is used
+/// to print transitions.
+pub const INFO_LOG: u32 = 1;
```

Comment:
Is there a reason this needs to be pub? Do the others also not need to be pub?

Reply:
This is the only level that's being used directly in the state machine impl. It's kinda sub-optimal, but it's helpful for stepping through the state machine exec

Reply:
there's a bit more context in this thread https://github.com/proptest-rs/proptest/pull/310#discussion_r1161370720